client/notices.go

```diff
@@ -181,6 +193,13 @@ func makeNoticesQuery(opts *NoticesOptions) url.Values {
 	if opts == nil {
 		return query
 	}
+	if len(opts.UserIDs) > 0 {
+		userIDStrs := make([]string, 0, len(opts.UserIDs))
+		for _, u := range opts.UserIDs {
+			userIDStrs = append(userIDStrs, strconv.FormatInt(int64(u), 10))
+		}
+		query["user-ids"] = userIDStrs
+	}
```

Comment:
Let's move this block down under Keys so the fields are handled in the same order as in `NoticesOptions` (as they are now).